src/HotChocolate/Fusion-vnext/src/Fusion.Composition/PostMergeValidationRules/EmptyMergedEnumTypeRule.cs

```diff
@@ -0,0 +1,32 @@
+using HotChocolate.Fusion.Events;
+using HotChocolate.Fusion.Events.Contracts;
+using HotChocolate.Fusion.Extensions;
+using static HotChocolate.Fusion.Logging.LogEntryHelper;
+
+namespace HotChocolate.Fusion.PostMergeValidationRules;
+
+/// <summary>
+/// TODO: Summary and spec link
```

Comment:
We can just assume that the spec URL fragment will be `#sec-Empty-Merged-Enum-Type`.